pkg/minishift/cache/oc_caching_test.go

```diff
@@ -79,9 +86,27 @@ func setUp(t *testing.T) {
 	testOc = Oc{"v1.3.1", filepath.Join(testDir, "cache")}
 }
 
-func EnsureGitHubApiAccessTokenSet(t *testing.T) {
-	if githubutils.GetGitHubApiToken() == "" {
-		t.Skip("Skipping GitHub API based test, because no access token is defined in the environment.\n " +
-			"To run this test check https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/ and set for example MINISHIFT_GITHUB_API_TOKEN (see github.go).")
+func addMockResponses(mockTransport *minitesting.MockRoundTripper) {
+	testDataDir := filepath.Join(basepath, "..", "..", "..", "test", "testdata")
+
+	mockTransport.RegisterResponse("https://api.github.com/repos/openshift/origin/releases/tags/v1.3.1", &minitesting.CannedResponse{
```

Comment:
Right. The whole point is to not go over the wire (or at least as little as possible). The testdata we are serving back in these tests (checksums, actual binaries, release info son) are pinned to when these tests got created. No need to update them. 

For what it's worths, I think you are referring to some refactoring in start_test.go. The change there is also cosmetic. These tests don't do any HTTP requests at all anymore.